apps/telemetry_event/lib/telemetry_event/helper/add_metrics.ex

```diff
@@ -2,14 +2,41 @@ defmodule TelemetryEvent.Helper.AddMetrics do
   @moduledoc false
 
   def add_metrics_options(metrics_options) do
-    [events_handled() | metrics_options]
+    metrics_options
+    |> events_handled()
+    |> phoenix_endpoint()
   end
 
-  defp events_handled() do
+  defp events_handled(metrics_options) do
     [
-      metric_name: "events_handled.count",
-      tags: [:app, :author, :dataset_id, :event_type],
-      metric_type: :counter
+      [
+        metric_name: "events_handled.count",
+        tags: [:app, :author, :dataset_id, :event_type],
+        metric_type: :counter
+      ]
+      | metrics_options
     ]
   end
+
+  defp phoenix_endpoint(metrics_options) do
+    case Application.get_env(:telemetry_event, :add_poller) do
+      true ->
+        [
+          [
+            metric_name: "phoenix.endpoint.stop.duration",
+            tags: [:request_path, :method],
+            tag_values: fn %{conn: conn} ->
+              %{request_path: Map.get(conn, :request_path), method: Map.get(conn, :method)}
+            end,
+            metric_type: :distribution,
+            unit: {:native, :millisecond},
+            reporter_options: [buckets: [0.01, 0.025, 0.05, 0.1, 0.2, 0.5, 1]]
```

Comment:
New metric type here. This should make a histogram of some sort I think? Make sure it's outputting in a way we can consume in dev.